### ISSUE TITLE
Is it possible to record the packets through NAT

### ISSUE THREAD

**Issue by zzgundam** [pmacct/pmacct#591]
I had made pmacct record packets that host sent or received,but packets that host forwarded(MASQUERADE) gone with out record.Is it possible to record them?

**Comment by paololucente** [comment]
What daemon are you using? Is it pmacctd, so libpcap-based, or uacctd, so NFLOG-based?

**Comment by zzgundam** [comment]
I used pmacctd,here is the config file.
```
pcap_interfaces_map: /etc/pmacct/pcap_interfaces_map.map
pcap_filter: tcp[tcpflags] == tcp-syn
tcpflags_encode_as_array: true
plugins: mysql
aggregate: src_mac,dst_mac,src_host,dst_host,src_port,dst_port,proto,tcpflags
sql_port:3306
sql_history: 10m
sql_history_roundoff: h
sql_table: acct_%Y%m%d
sql_table_schema: /etc/pmacct/createTable.txt
sql_db: pmacct
sql_table_version: 7
sql_refresh_time: 60
```

**Comment by zzgundam** [comment]
> What daemon are you using? Is it pmacctd, so libpcap-based, or uacctd, so NFLOG-based?

I used pmacctd,here is the config file.
```
pcap_interfaces_map: /etc/pmacct/pcap_interfaces_map.map
pcap_filter: tcp[tcpflags] == tcp-syn
tcpflags_encode_as_array: true
plugins: mysql
aggregate: src_mac,dst_mac,src_host,dst_host,src_port,dst_port,proto,tcpflags
sql_port:3306
sql_history: 10m
sql_history_roundoff: h
sql_table: acct_%Y%m%d
sql_table_schema: /etc/pmacct/createTable.txt
sql_db: pmacct
sql_table_version: 7
sql_refresh_time: 60
```

**Comment by jpds** [comment]
I've also recently discovered `pmacctd` and was looking for a solution to this too. I configured `pmacctd` to listen on both my WAN and LAN interface with a `pcap_interfaces_map`, which I then forward to an IPFIX collector on a different port.

At this IPFIX collector, I see flows like this:

```
2022-03-27 20:09:41.822 19389.140 UDP        172.20.1.x:35961 ->  185.206.y.z:3122  ........   0   904085  177.6 M       46    73259    196   494
2022-03-27 20:09:41.786 20271.549 UDP      100.64.p.q:35961 ->  185.206.y.z:3122  ........   0   856425  170.0 M       42    67098    198   330
```

Where the `172.20` is the internal address and `100.64` is how it's then NAT'ed out.

I collect the LAN port as I would like to see things like DHCP requests, and be able to track the outgoing flows to individual hosts themselves, not just the flows that are seen leaving the WAN port on the router.

However, so far I haven't found an easy way to deduplicate this traffic after trying various configs (v6 flows of course don't have this issue and work fine).
